### DISCUSSION TITLE
`error: 0-3:unknown keyword` from `regolith-look refresh` on 2.1

### DISCUSSION THREAD

**Discussion by robin-wayve** [regolith-linux/regolith-desktop#711] [Q&A] [answered]
```
> regolith-look refresh
Executing look loader script: /usr/share/regolith-look/default_loader.sh
> Updating already existing 'Regolith' profile
error: 0-3:unknown keyword

Usage:
  dconf write KEY VALUE 

Write a new value to a key

Arguments:
  KEY         A key path (starting, but not ending with '/')
  VALUE       The value to write (in GVariant format)
```

I am just using the default look and not much else custom: I ran this because I noticed my i3xrocks not showing any more (a reboot seems to have fixed that issue though).

Diagnostic here: https://gist.github.com/robin-wayve/6691096ab45276150e6d13a09c802f7c

Recently upgraded from 1.6.

**Comment by kgilmer** [comment, marked as answer]
Interesting, and the "error: 0-3:unknown keyword" problem persists after the reboot?  I reviewed the diagnostic output and didn't find anything suspect.  The error seems to be coming from the [gnome-terminal profile creation script](https://github.com/regolith-linux/regolith-ftue/blob/main/regolith-init-term-profile).  I've searched your Xresource values but didn't find anything suspect.  If you want to troubleshoot further you may want to copy that script to `/tmp` and add `set -x` at the top, and then run it to see which line is the source of the problem.

**Reply by robin-wayve** [reply]
It looks like it was because I had `gnome.terminal.use-transparent-background: yes` instead of `true`. 🤦🏻 Thanks for the tip!
